requirements-python.txt

```diff
@@ -1,3 +1 @@
-cbor2==4.1.2
```

Comment:
I don't know what the cbor dependency here is, but it doesn't seem necessary for the Civis Python client to work.  pubnub is no longer needed since the Python client doesn't use it anymore.